charts/nango/values.yaml

```diff
@@ -38,6 +38,7 @@ server:
   useLoadBalancer: true
   MAILGUN_API_KEY: ""
   replicas: 1
+  SERVER_PORT: 3306
```

Comment:
Two things:

1. This should be set as: `SERVER_PORT: 3003`
2. We can/should have _some_ way of ordering these variables - the convention within the file appears to be to group the `UPPERCASE` (environment variables) together - so let's do that and move them to the bottom of the "server" block and then alphabetize both groups (lowercase/uppercase).

Reply:
Not sure why I put that port value, but it's fixed.

I didn't initially rearrange/alphabetize the vars because I didn't see pattern elsewhere - the only other section I see that has both uppercase and lowercase var names, the uppercase vars are sandwiched between two groups of lowercase vars (the last group is only one var so it's easy to miss). But the change has been made.